.test-infra/jenkins/job_LoadTests_GBK_Java.groovy

```diff
@@ -33,7 +33,7 @@ def loadTestConfigurations = { mode, isStreaming ->
                             appName               : "load_tests_Java_Dataflow_${mode}_GBK_1",
                             tempLocation          : 'gs://temp-storage-for-perf-tests/loadtests',
                             publishToBigQuery     : true,
-                            bigQueryDataset       : 'load_test',
+                            bigQueryDataset       : loadTestsBuilder.setBQDatasetNameByTriggerContext('load_test', context),
```

Comment:
Can we pass the datasetName same as we pass `mode` and `isStreaming` parameters? It will allow us not to call the static method multiple times.

Reply:
Why not. We won't have to pass `context` parameter too. Thanks for this tip.